scripts/highlighting.js

```diff
@@ -183,80 +183,95 @@ async function FilesInPrBitbucket(response) {
  * @param {Array} apiResponses - Array of API responses containing hunk information.
  */
 const githubHunkHighlight = async (apiResponses) => {
+	// checking for diff view either unified or split
+	let isUnifiedDiffView = false;
+	const allInputTagsWithValueAttr = document.querySelectorAll('input[value]'); // for reducing DOM queries in foreach loop.
+	for (const item of allInputTagsWithValueAttr) {
+		const valueAttrOfInputTag = item.getAttribute('value');
+		const checkedAttrOfInputTag = item.getAttribute('checked');
+
+		if (valueAttrOfInputTag === 'unified' || valueAttrOfInputTag === 'split') {
+			if (checkedAttrOfInputTag === 'checked' && valueAttrOfInputTag === 'unified') {
+				isUnifiedDiffView = true; // for unified view
+			}
+		}
+	};
```

Comment:
could have done in a never nesting way, LOW LOGAF

Reply:
Yes, I could have made this a single conditional by removing the first condition, but I have kept it to show the thought process in the code: we first find the input field that has the relevant information and then choose the value.

I don't think never nesting is actually possible here. There is no error condition here. It is an if-else system.